tools/topology/platform/intel/intel-generic-dmic.m4

```diff
@@ -20,13 +20,13 @@ dnl     frames, deadline, priority, core)
 # Schedule 48 frames per 1000us deadline on core 0 with priority 0
```

Comment:
@juimonen Comments need to be adjusted in every topology.

Reply:
ok